scripts/monorepo/getDependencies.js

```diff
@@ -14,10 +14,12 @@ function flattenPackageGraph(rootPackages, projectGraph, packageList = []) {
 }
 
 /**
- * Returns all the dev dependencies of a given package name
+ * Returns all the dependencies of a given package name
  * @param {string} packageName including `@fluentui/` prefix
+ * @param {String} options.dev include dev dependencies
+ * @param {String} options.production include production dependnecies
  */
-async function getDevDependencies(packageName) {
+async function getDependencies(packageName, options = {}) {
```

Comment:
I think options should be required since otherwise the function will return nothing. Either that or default one or more of `dev` or `production` to true.

Reply:
```suggestion
async function getDependencies(packageName, options = { production: true }) {
```